2025-07-05 22:07:00,973 - urllib3.connectionpool - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ConnectionResetError(10054, 'An existing connection was forcibly closed by the remote host', None, 10054, None)': /session/529989f0ebc834887bc6b611c65ab063
2025-07-05 22:07:05,055 - urllib3.connectionpool - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001F3791D4F90>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/529989f0ebc834887bc6b611c65ab063
2025-07-05 22:07:09,230 - urllib3.connectionpool - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001F3791EC390>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/529989f0ebc834887bc6b611c65ab063
